src/cursor/abstract_cursor.ts

```diff
@@ -767,15 +781,26 @@ export abstract class AbstractCursor<
         'Unexpected null selectedServer. A cursor creating command should have set this'
       );
     }
+    const getMoreOptions = {
+      ...this.cursorOptions,
+      session: this.cursorSession,
+      batchSize,
+      omitMaxTimeMS: this.cursorOptions.omitMaxTimeMS
+    };
+
+    if (
+      this.cursorOptions.tailable &&
+      this.cursorOptions.awaitData &&
+      this.cursorOptions.maxAwaitTimeMS != null
+    ) {
+      getMoreOptions.maxTimeMS = this.cursorOptions.maxAwaitTimeMS;
+    }
```

Comment:
```suggestion
```

GetMoreOperation looks for [maxAwaitTimeMS, not maxTimeMS](https://github.com/mongodb/node-mongodb-native/blob/681ddd8d539f77a2db80c7b9af3710c63a1b112c/src/operations/get_more.ts#L87).  This likely is passing because you spread `this.cursorOptions` into getMoreOptions already, so `maxAwaitTimeMS` is provided to the GetMoreOperation.